practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -109,12 +112,11 @@ public PractitionerDetails getPractitionerDetails(
             IBaseResource practitioner =
                     practitioners.size() > 0 ? practitioners.get(0) : new Practitioner();
             Long practitionerId = null;
-            String practitionerIdStringValue = null;
+            String practitionerIdStringValue;
             boolean isIdNumeric = false;
             if (practitioner.getIdElement() != null
                     && practitioner.getIdElement().getIdPart() != null
-                    && StringUtils.isNumeric(
-                            ((IBaseResource) practitioner).getIdElement().getIdPart())) {
+                    && StringUtils.isNumeric(practitioner.getIdElement().getIdPart())) {
                 practitionerId =
                         practitioner.getIdElement() != null
                                         && practitioner.getIdElement().getIdPart() != null
```

Comment:
We already have the null check in the outer condition as well as the default of `0L`, we can refactor this line to just be   `practitionerId = practitioner.getIdElement().getIdPartAsLong();`